client/frontend/client.go

```diff
@@ -37,7 +37,7 @@ import (
 
 const (
 	// DefaultTimeout is the default timeout used to make calls
-	DefaultTimeout = 10 * time.Second
+	DefaultTimeout = time.Second * 10
```

Comment:
Can you flip it back (here and in other places)? It is more natural to represent "10 seconds" as `10 * time.Second`.